### PR TITLE
rollup-plugin-bundle-html

### PR DESCRIPTION
**Pull request by frederikhors**
Awesome Contribution Checklist:

- [x] I have read, and re-read the [Contributing Guidelines](https://github.com/rollup/awesome/blob/master/.github/CONTRIBUTING.md)
- [x] I have searched to ensure the suggested item doesn't exist on this list
- [x] This PR contains only one item

### CODE REVIEW

**Comment by shellscape** [comment]
Thanks for your submission :beer:

Can you summarize how this plugin differs from https://github.com/bengsfort/rollup-plugin-generate-html-template?

**Comment by frederikhors** [comment]
> Thanks for your submission 🍺
> 
> Can you summarize how this plugin differs from https://github.com/bengsfort/rollup-plugin-generate-html-template?

A lot. It injects css and externals automagically. Does hashing and more. Read yourself here: https://github.com/haifeng2013/rollup-plugin-bundle-html#rollup-plugin-bundle-html

**Comment by shellscape** [comment]
Appreciate the reply, but looking for more information. For addition to the list, the onus is on the submitter to respond to follow-up questions. Unfortunately, I don't have the time to deeply research each and every submission. So we ask that authors and submitters take on that task.

**Comment by frederikhors** [comment]
Maybe the author, @haifeng2013, can answer better than me.
